### DISCUSSION TITLE
FS22 courseplay issues

### DISCUSSION THREAD

**Discussion by erniefehr** [Courseplay/Courseplay_FS22#3605] [Q&A]
When I'm planting with multiple planters, I am not able to get any planting overlap on headlands, thus always leaving gaps on headlands. My settings in Raise Tool are set to "late" and Lift Tool is set to "early". Not sure what to do to correct it, but any help would be appreciated.

**Comment by Tensuko** [comment]
Screenshots could help.

**Reply by erniefehr** [reply]
![IMG_1954](https://github.com/user-attachments/assets/e646b70d-7f33-4289-b660-d474f67adafe)

**Reply by Tensuko** [reply]
Thats an aligment issue.
We know that this can happen.
We even tried a calculation to prevent that.
There is nothing you can change for that.
We will find a solution for this in FS25, sry.

**Reply by Tensuko** [reply]
Actually, the stripe is a bit strange.
That could be helped with raise late and lower early.
